view.rb

```diff
@@ -1,74 +1,94 @@
+		print "\nWelcome to RubyBank, where all of your financial dreams come true."
+		print "\n\nWould you like to make an account? ('yes' or 'no'): " 
 		gets.chomp.downcase
 	end
 
-	### Says goodbye and mourns the loss of user.
-	def self.goodbye 
-		puts "\nThat is too bad. We are a great bank. Suit yourself! Have a nice day!\n"
-	end
-	### Explains that method is not yet available, check back later.
-	def self.goodbye2			
-		puts "\nThis is not yet a current working method in RubyBank. Check back in a few hours.\nGoodbye!\n\n"
+	def self.goodbye # Says goodbye and mourns the loss of user.
+		print "\nThat is too bad. We are a great bank. Suit yourself! Have a nice day!\n\n"
 	end
 
 	def self.goodbye_cust # The goodbye message for a customer. 
-		puts "\nHave a great day! We will look forward to your return!\n"
+		print "\nHave a great day! We will look forward to your return!\n\n"
 	end
 
 	def self.account_name # this should be the dialog around initializing your bank account. You give your account name.	
-		puts "\nplease enter your the name you wish to associate with the account."
+		print "\nPlease enter your the name you wish to associate with the account: "
 		gets.chomp.downcase
 	end
 
-	def self.account_balance # this should be the dialog around initializing your bank account. You give your account password.	
-		puts "\nplease enter the initial balance you would like to deposit into your new account."
-		gets.chomp.downcase.to_i
-	end
+  def self.signin_name
+    print "\nPlease enter your account name: "
+    gets.chomp.downcase
+  end
+
+  def self.signin_pin
+    print "\nPlease enter your account pin: "
+    gets.chomp.downcase.to_i
+  end
+
+  def self.existing_account_error
+    print "\n Our records inticate that you already have an account with us. Please sign in to your account.\n"
+  end
 
 	def self.how_can_we_help_you # Dialog around asking user to enter their choice.
-		puts "\nHow can we help you? (please enter 'deposit', 'withdraw', 'balance', or 'leave')"
-		gets.chomp.downcase
+		print "\nHow can we help you? Would you like to 'deposit' or 'withdraw' to/from your account, view your current 'balance', or 'end' your session?"
+		print "\n\n(Please enter 'deposit', 'withdraw', 'balance', or 'end'): "
+		gets.chomp.downcase 
+	end
+
+	def self.space
+		print "\n\n\n"
 	end
 
-	def self.display_balance #Should be dialog around user request to check balance.
-		print "\nYour current balance is: $ "
+	def self.account_prompt
+		puts "Your total accounts: " 
 	end
 
-	def self.withdraw #Should be dialog around user request to withdraw money from their account.
-		puts "'\nPlease enter the amount you would like to withdraw."
+	def self.pick_your_account
+		print "\n\n\nYour account number is displayed above. Please enter your account number: "
+		gets.chomp.downcase.to_i
+	end 
+
+	def self.withdraw # Should be dialog around user request to withdraw money from their account.
```

Comment:
Maybe you should delete this method because it doesn't seem like you're using it.